.php-cs-fixer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$finder     = Finder::create()
+    ->in(__DIR__)
+    ->name('*.php');
+
+$config = new Config();
+
+$rules = [
+    '@PER-CS2.0'                      => true,
+    'array_indentation'               => true,
+    'array_syntax'                    => ['syntax' => 'short'],
+    'binary_operator_spaces'          => ['default' => 'align_single_space_minimal'],
+    'blank_line_after_namespace'      => true,
+    'blank_line_after_opening_tag'    => true,
+    'blank_line_before_statement'     => ['statements' => ['return']],
+    'braces'                          => ['allow_single_line_closure' => true],
+    'cast_spaces'                     => ['space' => 'single'],
+    'class_attributes_separation'     => ['elements' => ['method' => 'one']],
+    'concat_space'                    => ['spacing' => 'one'],
+    'declare_equal_normalize'         => ['space' => 'single'],
+    'function_typehint_space'         => true,
+    'lowercase_cast'                  => true,
+    'no_whitespace_in_blank_line'     => true,
+    'single_blank_line_at_eof'        => true,
+    'single_quote'                    => true,
+    'space_after_semicolon'           => true,
+    'trailing_comma_in_multiline'     => ['elements' => ['arrays']],
+    'trim_array_spaces'               => true,
+    'unary_operator_spaces'           => true,
+    'whitespace_after_comma_in_array' => true,
```

Comment:
Are these additional rules on top of PER2 ?
I think some things like the `single_blank_line_at_eof` rule already part of PSR.

Also, I think one of the goal of the PSR/PHP standard is to enforce common rules for all PHP app.
Having extra rules on top of that might be counter intuitive, no ?

Reply:
Alright, I'll remove the other rules. However, I'll keep trailing_comma_in_multiline, because without this setting, the fix command also adds commas in functions, which I find a bit confusing.

Reply:
They feel a bit confusing at first but the trailing comas are supported by most programming languages because they have a few minor advantages:
* Better for version control as you don't need to modify the last line if you add a new value after it (smaller diff, better git blame and less conflicts risks)
* Lets you rearrange the items by moving lines around using your IDE shortcuts without having to worry about having to add a comma to an element when it moves out of the final position, or removing a comma from the element that moved into the final position.

Reply:
The CI doesn't seem to agree:

![image](https://github.com/user-attachments/assets/2d64f997-f9f5-47ff-8ee6-e0220fc59386)

Reply:
Did you execute the command on PHP 8 ? 
PHPCS fixer has different behavior depending if you run it on PHP 7.4 or PHP 8.

Since plugins must be compatible with PHP 7.4, I guess PHPCS fixer should also be ran using PHP 7.4 :/

Reply:
Indeed, the results differ since PHP 7.4, but it’s inconvenient to switch PHP versions just to run the fix command.